src/command-registry.js

```diff
@@ -198,23 +200,20 @@ module.exports = class CommandRegistry {
         listeners = this.inlineListenersByCommandName[name]
         if (listeners.has(currentTarget) && !commandNames.has(name)) {
           commandNames.add(name)
-          commands.push({ name, displayName: _.humanizeEventName(name) })
+          const targetListeners = listeners.get(currentTarget);
+          commands.push(
+            ...targetListeners.map(listener => listener.getPublicCommand())
```

Comment:
@nathansobo not really sure what the best thing to do in the case of multiple matching commands for the same element and commandName. 

I'm matching the behavior below in selector-based listeners, which means returning all matching commands, but am concerned about multiple listings (each with their own metadata) in e.g. command palette. 

Even in that case, choosing only one of the matching commands would have the effect or running every single listener :'(